liquidation-monitor/src/agent.js

```diff
@@ -416,8 +451,12 @@ function provideHandleBlock(data) {
           liquidationAmount.dp(2).toString(),
           shortfallUSD.dp(2).toString(),
           accounts[currentAccount].health.dp(2).toString(),
+          blockEvent.block.timestamp,
+          data,
         );
-        findings.push(newFinding);
+        if (newFinding != null) {
```

Comment:
```suggestion
        if (newFinding !== null) {
```

Reply:
This is intentional. `x != null` is the same as `x !==null || x !==undefined` I do want to match both of those. `undefined` for when no findings are found and `null` for findings that are filtered. I've made it explicit in commit e6c3b366a04c0684716915aae00171dec08e9b6e